client/src/features/dashboardV2/DashboardV2.tsx

```diff
@@ -567,16 +567,17 @@ function ViewAllLink({
   noItems: boolean;
   total: number;
 }) {
+  const searchUrl = ABSOLUTE_ROUTES.v2.search;
   return noItems ? (
     <Link
-      to={`/v2/search?page=1&perPage=12&q=type:${type}`}
+      to={`${searchUrl}?page=1&perPage=12&q=type:${type}`}
```

Comment:
Do not specify `page` and `perPage`, these are defined from defaults in the search page. And prefer to separate the path from the search:

```suggestion
      to={{ pathname: searchUrl, search: `q=type:${type}` }}
```

Also, we should use a search query helper to avoid hard-coding constants (they are defined in `searchV2.constants.ts` -> `type`, `:`, etc.). We can leave this for later, as the `buildSearchQuery()` function is not friendly to use outside of `searchV2.slice.ts`.

Reply:
Fixed in df3eff7347aa71602b79e05e4e83c82160f1e43c